### DISCUSSION TITLE
Container health check

### DISCUSSION THREAD

**Discussion by JohnyboyOlazar** [itzg/docker-minecraft-server#998] [Q&A]
Hi there, I was wondering if I can change the container health checker to the custom settings that I have. My example I changed the port the server runs on and would like to be able to let the health check run on that port then as well. Is there a config where I can set the server port for the health checker?

![image](https://user-images.githubusercontent.com/71178265/128636423-bcf7246a-832f-4df8-8424-0e53d82153ea.png)

**Comment by itzg** [comment]
The health check script uses the variable `SERVER_PORT`...but I would question why you're changing the server port since Docker port mappings solves for port conflicts.

**Reply by JohnyboyOlazar** [reply]
I like to change the default port that the server runs on cause of odd activity I've noticed in the Firewall of my Server where there have been constant requests coming in from unknown IP addresses which I just wanted to avoid by using a port that is not the default one.

**Reply by itzg** [reply]
Ok, that's what docker port mappings are for. Again, why are you changing the container-internal port number?
